tests/v2/calculator.test.ts

```diff
@@ -1,10 +1,12 @@
+/// <reference types="jest" />
 'use strict';
 
-const assert = require('assert');
-const { Calculator } = require('../../src/calculator/v2');
-const IIIFError = require('../../src/error');
-
-const { v2: { qualities, formats, regions, sizes, rotations } } = require('../fixtures/iiif-values');
+import { describe, it, beforeEach } from '@jest/globals';
+import assert from 'assert';
```

Comment:
Not necessary for this PR, but since you're already using `assert`, you might be interested in [Node's native test runner](https://nodejs.org/api/test.html), which is significantly faster than `jest` and doesn't require the dependency.

Reply:
Sure thing. I wasn't thinking you'd do that with this large PR anyway. Probably better to keep this focused 👍